Tests/App/bootstrap.php

```diff
@@ -1,9 +1,12 @@
 <?php
 
 use Doctrine\Common\Annotations\AnnotationRegistry;
+use Symfony\Bridge\PhpUnit\DeprecationErrorHandler;
 
 $loader = require __DIR__.'/../../vendor/autoload.php';
 
 AnnotationRegistry::registerLoader(array($loader, 'loadClass'));
 
+DeprecationErrorHandler::register(getenv('SYMFONY_DEPRECATIONS_HELPER'));
```

Comment:
you don't need it. The deprecation error handler is registered automatically when calling phpunit and the bridge is installed through composer